### DISCUSSION TITLE
last.fm scrobbling?

### DISCUSSION THREAD

**Discussion by psoleckimoj** [losnoco/Cog#391] [Q&A]
Hi,
New to Cog and liking it so far, it mentions last.fm support. I try and scrobble everything but can't figure out where it enable it/enter my details?
Cheers,
Paul.

**Comment by may-iis** [comment]
Hello!
I am also really interested in having the last.fm scrobbler included. Unfortunately I'm no actual programmer but maybe with some hints it would'nt be too hard to code it.
Best,
Tom
